_styles/main.css

```diff
@@ -379,6 +416,37 @@ nav .logomark:focus .logomark-svg {
   }
 }
 
+@media (min-width: 800px) {
```

Comment:
Can we set this to something lower? I know that 800 is where the view switches to having more width but even down to 650px everything fits fine on a tablet sized view.

Reply:
I've now set it to 680px.